Cubical/Data/Int/MoreInts/QuoInt/Properties.agda

```diff
@@ -233,3 +236,84 @@ sign-pos-suc-· m (signed s (suc n)) = refl
 
 ·-injʳ : ∀ m n o → m · pos (suc o) ≡ n · pos (suc o) → m ≡ n
 ·-injʳ m n o p = ·-injˡ o m n (·-comm (pos (suc o)) m ∙ p ∙ ·-comm n (pos (suc o)))
+
+private
+  _+'_ : ℤ → ℤ → ℤ
+  _+'_ = Iso.fun (binaryOpIso isoIntℤ) Int._+_
+
+  sucℤ→Int : ∀ (n : ℤ) → Int.sucℤ (ℤ→Int n) ≡ ℤ→Int (sucℤ n)
+  sucℤ→Int (pos n) = refl
+  sucℤ→Int (neg zero) = refl
+  sucℤ→Int (neg (suc zero)) = refl
+  sucℤ→Int (neg (suc (suc n))) = refl
+  sucℤ→Int (posneg i) = refl
+
+  predℤ→Int : ∀ (n : ℤ) → Int.predℤ (ℤ→Int n) ≡ ℤ→Int (predℤ n)
+  predℤ→Int (pos zero) = refl
+  predℤ→Int (pos (suc n)) = refl
+  predℤ→Int (neg zero) = refl
+  predℤ→Int (neg (suc n)) = refl
+  predℤ→Int (posneg i) = refl
+
+  ℤ→Int+Int≡+ : ∀ (n m : ℤ) → (ℤ→Int n) Int.+ (ℤ→Int m) ≡ ℤ→Int (n + m)
+  ℤ→Int+Int≡+ n m = (ℤelim (λ n → ∀ (m : ℤ) → (ℤ→Int n) Int.+ (ℤ→Int m) ≡ ℤ→Int (n + m)) posℤ→Int+Int≡+ negsucℤ→Int+Int≡+ n) m
```

Comment:
I think a name containing something like 'IsHom+' would be better.
And then this fact should be exported as well (maybe even as an homomorphism of abelian groups - that might fit in better somewhere else though).

Reply:
Just to be clear: The equality you proved is exactly the statement that the map ```ℤ→Int``` is a homomorphism with respect to ```+```. As you suspect, the performance of a homomorphism proof extracted from an equality of groups is not going to be great, so we should export this one separately.
Regarding conventions, it would be better to turn the equality around. In general, we try to write equations like this in a "evaluating/simplifying/normalizing direction" (meaning ```f(x+y)=f(x)+f(y)``` for a homomorphism or ```x*(y+z)=x*y+x*z``` for distributivity). This is not a well defined term and there is no reason to do it this way -- it is just about having a convention people can remember.

Reply:
Right, the thing I wrote about "iso-" was just silly, sorry. Anyway, exported, renamed, and flipped.

I also rebased this, but in the meantime another thing got merged so that was a tad pointless.